test/unit/api/cpp/datatype_api_black.cpp

```diff
@@ -274,6 +274,8 @@ TEST_F(TestApiBlackDatatype, parametricDatatype)
   Sort pairType = d_solver.mkDatatypeSort(pairSpec);
 
   ASSERT_TRUE(pairType.getDatatype().isParametric());
+  std::vector<Sort> dparams = pairType.getDatatype().getParameters();
+  ASSERT_TRUE(dparams[0] == t1 && dparams[1] == t2);
```

Comment:
Add `ASSERT_THROW` test, too (for the case when it is not a parametric datatype). And what about the case when it is an instantiated datatype?